ui/app/models/mount-config.js

```diff
@@ -33,8 +33,11 @@ export default class MountConfigModel extends Model {
   auditNonHmacResponseKeys;
 
   @attr('mountVisibility', {
-    editType: 'boolean',
-    label: 'List method when unauthenticated',
+    label: 'Use as preferred UI login method',
+    editType: 'toggleButton',
+    helperTextEnabled: 'Shown as the default login method on the unauthenticated UI login endpoint.',
+    helperTextDisabled:
+      'Turn on the toggle to use this auth mount as the preferred login method during UI login.',
```

Comment:
small copy change for accuracy (no need to run by design)
```suggestion
      'Turn on the toggle to use this auth mount as a preferred login method during UI login.',
```